config/site.yml.tmpl

```diff
@@ -25,6 +25,11 @@ secure_cookies: false
 #
 secret_token: "change-me"
 
+# By default, tell Rails to not serve static assets. The best practice is to
```

Comment:
Yeah, the message is good, it's just a style suggestion that the first line be only a summary of what the option _does_ rather than a description what it's _set to_. For example, "Serve static assets (images, stylesheets, scripts) in production," followed by an explanation of the defaults, etc., on subsequent lines.